packages/headless/src/user.ts

```diff
@@ -90,6 +90,50 @@ export function useUpdateUser({
   });
 }
 
+export function useUpdateUserPersonalInfo({
+  onSuccess,
+  onError,
+}: {
+  onSuccess?: OnSuccess<IUser.Self>;
+  onError?: OnError;
+}) {
+  const atlas = useAtlas();
+  const update = useCallback(
+    async ({ id, payload }: { id: number; payload: IUser.UpdateApiPayload }) =>
+      atlas.user.update(id, payload),
+    [atlas.user]
+  );
+
+  return useMutation({
+    mutationFn: update,
+    mutationKey: [MutationKey.UpdateUserPersonalInfo],
+    onSuccess,
+    onError,
+  });
+}
+
+export function useUpdateTutorTopics({
```

Comment:
Topics are not specific for tutors. Students can also update their topics selection. This why we should call this hook `useUpdateUserTopics`. 

```suggestion
export function useUpdateUserTopics({
```